src/main/java/net/solmis/cityparking/requests/GetParkingReceiptRequest.java

```diff
@@ -0,0 +1,38 @@
+package net.solmis.cityparking.requests;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.util.ArrayList;
+
+public class GetParkingReceiptRequest {
+
+    public static String DRIVER_DISABLED = "disabled";
+    public static String DRIVER_REGULAR = "regular";
+
+    private static ArrayList<String> driverTypes;
+
+    @Positive(message="Attribute ticketId should be a positive number.")
+    public long ticketId;
+
+    @NotNull(message="Attribute driverType is required.")
+    public String driverType;
+
+    @JsonCreator
+    public GetParkingReceiptRequest(@JsonProperty("ticketId") final long ticketId,
+                                    @JsonProperty("driverType") final String driverType) {
+        this.ticketId = ticketId;
+        this.driverType = driverType;
+    }
+
+    public static ArrayList<String> getDriverTypes() {
```

Comment:
dlaczego ta klasa ma takie informacje?

Reply:
Długo się zastanawiałem gdzie to umieścić i zgodzę się, że nie jest to oczywiste miejsce do szukania takich informacji. Nie wiem, gdzie można by to umieścić. W osobnej malutkiej klasie? W jakimś miejscu, gdzie są zebrane różne dane konfiguracyjne?